api-reference/10 UI Components/dxValidator/8 Validation Rules/PatternRule/PatternRule.md

```diff
@@ -15,5 +15,7 @@ To specify the regular expression that the validated field must match, set the r
     href: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/Validation/Overview/"
 }
 
+[note] If you apply a pattern rule to a DateBox component, specify [dateSerializationFormat](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#dateSerializationFormat) so the format of validated value is a string.
```

Comment:
```suggestion
[note] If you apply a pattern rule to a DateBox component, specify [dateSerializationFormat](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#dateSerializationFormat) so that the format of the validated value is a string.
```